Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -1008,6 +1009,35 @@ public protocol PurchasesSwiftType: AnyObject {
 
     #endif
 
+    /**
+     * Use this method only if you already have your own IAP implementation using StoreKit 2 and want to use
+     * RevenueCat's backend. If you are using StoreKit 1 for your implementation, you do not need this method.
+     *
+     * You only need to use this method with *new* purchases. Subscription updates are observed automatically.
+     *
+     * #### Example:
+     *
+     * ```swift
+     * let result = try await product.purchase(options: options)
+     * Purchases.shared.handleObserverModeTransaction(purchaseResult: result)
+     * ```
+     *
+     * - Note: You need to finish the transaction yourself after calling this method.
+     *
+     * - Parameter purchaseResult: The ``StoreKit.Product.PurchaseResult`` of the product that was just purchased.
```

Comment:
I don't think you can reference types outside the module with "``", does this not give a warning when building?

Reply:
I haven't noticed any warning, but replaced with single ticks since it cannot link anywhere ayway.

Reply:
![Screenshot 2024-01-17 at 12 18 25](https://github.com/RevenueCat/purchases-ios/assets/685609/df3283c1-620e-453f-98b6-360d3e654c02)